mappings/net/minecraft/block/CandleCakeBlock.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5545 net/minecraft/block/CandleCakeBlock
+	FIELD field_27183 LIT_2 Lnet/minecraft/class_2746;
```

Comment:
```suggestion
	FIELD field_27183 LIT Lnet/minecraft/class_2746;
```

Reply:
Fixed in 48c14bf52cc9898db151ecdd2b02c0851839a51a